socorro/app/socorro_app.py

```diff
@@ -145,8 +146,37 @@ def _app_instance_name(self):
             os.getpid(),
         )
 
+    @classmethod
+    def configure_sentry(cls, basedir, host_id, sentry_dsn):
+        release = get_release_name(basedir)
+        scrubber = Scrubber(rules=SCRUB_RULES_DEFAULT)
+        set_up_sentry(
+            sentry_dsn=sentry_dsn,
+            release=release,
+            host_id=host_id,
+            # Disable frame-local variables
+            with_locals=False,
+            # Disable request data from being added to Sentry events
+            request_bodies="never",
+            # All integrations should be intentionally enabled
+            default_integrations=False,
+            integrations=[
+                AtexitIntegration(),
+                Boto3Integration(),
+                ExcepthookIntegration(),
+                DedupeIntegration(),
+                StdlibIntegration(),
+                ModulesIntegration(),
+                ThreadingIntegration(),
+            ],
+            # Scrub sensitive data
+            before_send=scrubber,
+        )
```

Comment:
App gets subclassed into the actual apps. To reduce duplicate code and the potential to fix things in one place but miss others, we should nix this and make it required to implement on an app-by-app basis.